pandas/_testing/_io.py

```diff
@@ -82,10 +82,7 @@ def dec(f):
         is_decorating = not kwargs and len(args) == 1 and callable(args[0])
         if is_decorating:
             f = args[0]
-            # pandas\_testing.py:2331: error: Incompatible types in assignment
-            # (expression has type "List[<nothing>]", variable has type
-            # "Tuple[Any, ...]")
-            args = []  # type: ignore[assignment]
+            args = ()
```

Comment:
this should be able to just be removed entirely

Reply:
hmm, we call `dec` with `f` and it uses the `args` from the outer scope. I assume that zeroing `args` out is intentional?